### DISCUSSION TITLE
Editing basic text files within GitHub Mobile app

### DISCUSSION THREAD

**Discussion by CrazyContraption** [community/community#18369] [Mobile]
Is there any way that we can implement a feature for editing basic file contents using the GitHub Mobile app?

Making small corrections to typos or being able to write mundane description files on the go so we can be ready to merge/pull when on site would be amazing.

**Comment by GuScarpim** [comment]
You can't edit from the mobile version of GitHub website. You have to switch to desktop site and then edit. In mobile version of GitHub site, the editor is not available. Alternatively, you can replace "blob" with "edit" in the mobile URL
